extensions/azurePublish/src/components/azureProvisionDialog.tsx

```diff
@@ -747,43 +692,39 @@ export const AzureProvisionDialog: React.FC = () => {
     }
   }, [listItems]);
 
-  const selection = useMemo(() => {
-    const s = new Selection({
-      onSelectionChanged: () => {
-        const list = s.getSelection();
-        setEnabledResources(list);
-      },
-      canSelectItem: (item, index) => {
-        return item.required === false;
-      },
-    });
-    if (s && listItems) {
-      s.setItems(listItems, false);
-      s.setAllSelected(true);
-    }
-    return s;
-  }, [listItems]);
+  const PageAddResources = () => {
```

Comment:
this is like a component defined within another component. why not extract it to it's own component?

Reply:
That will happen eventually.